test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -522,10 +520,8 @@ def set_vntype(self, mom, typestring='myvntype'):
         vntype_file = os.path.join(pbs_home, 'mom_priv', 'vntype')
         self.logger.info("Setting vntype to %s in %s on mom %s" %
                          (typestring, vntype_file, mom))
-        (fd, fn) = self.du.mkstemp()
-        os.write(fd, typestring)
-        os.close(fd)
-        ret = self.du.run_copy(hosts=mom, src=fn,
+        fn = self.du.create_temp_file(body=typesting)
+        ret = self.du.run_copy(hosts=self.mom.hostname, src=fn,
```

Comment:
You mean 'mom' instead 'self.mom.hostname' right? Also I know it's not your changes but I suggest to use uid and gid == 0 instead 'root' name...